frontend/components/propositions/OccurrenceDetail.tsx

```diff
@@ -36,7 +37,10 @@ const OccurrenceDetail: FC<OccurrenceDetailProps> = ({ occurrence }: OccurrenceD
           )
       )}
 
-      <h2 className="text-center my-3" dangerouslySetInnerHTML={{ __html: occurrence.title }} />
+      <h2
+        className="text-center my-3"
+        dangerouslySetInnerHTML={{ __html: titleCase(occurrence.title) }}
+      />
       {occurrence.summary != occurrence.title && <p className="lead">{occurrence.summary}</p>}
```

Comment:
@senamn, can you change this line so that the `p` element uses `dangerouslySetInnerHTML` for `occurrence.summary`? 

That will fix the <em> tags being displayed around "Homo sapiens."

Reply:
> @senamn, can you change this line so that the `p` element uses `dangerouslySetInnerHTML` for `occurrence.summary`?
> 
> That will fix the `<em>` tags being displayed around "Homo sapiens."

Yes, fixed it accordingly and will be pushing the updated code!